source/Generate/Type.elm

```diff
@@ -50,6 +50,9 @@ renderType definition =
             Array_ items ->
                 typeAliasDecl <| list <| renderPropertyType name "Item" <| getItemsType items
 
+            Dict_ typeName ->
+                typeAliasDecl <| dict <| renderPropertyType name "Item" typeName
```

Comment:
Use `type_` or similar since it refers to a `Type`. `typeName` would shadow the imported function.